bin/publish-pacts.js

```diff
@@ -27,9 +26,9 @@ async function removePactsNotToBePublished () {
     .forEach(async filename => await unlink(pactDirPath + filename))
 }
 
-async function run() {
-  //make sure pact dir exists, and clean out any files marked to be
-  if (existsSync(pactDirPath)) {
+async function run () {
+  // make sure pact dir exists, and clean out any files marked to be
+  if (await stat(pactDirPath)) {
```

Comment:
Very minor but the interface on these isn't quite the same, stat with throw an `ENOENT` exception very similar to POSIX `stat`, because of this the console output will never be reach, functionally equivalent might be

```
try {
  await stat(pactDirPath)
  await removePactsNotToBePublished()
  await publish()
} catch (e) {
  console.log('Unable to publish pacts', e)
}
```

Or 

```
async function exists(path) {
  try {
    await stat(path)
    return true
  } catch (e) {
    return false
  }
}

async function run() {
  if (await exists(pactDirPath)) //...
}
```

Because this is a one time run synchronous script it just throwing the stat error and ending the process seems like a fine behaviour so don't think this needs to be changed, it might just be confusing to someone later